src/ray/gcs/gcs_server/gcs_task_manager.h

```diff
@@ -111,13 +125,21 @@ class GcsTaskManager : public rpc::TaskInfoHandler {
     /// replaced task event.
     absl::optional<rpc::TaskEvents> AddOrReplaceTaskEvent(rpc::TaskEvents &&task_event);
 
-    /// Get task events.
+    /// Get task events from job.
+    ///
+    /// Get task events from a job, with job_id. Additionally, a set of TaskID strings
```

Comment:
can you just add docstring to each signature?

Reply:
haha yeah - i was going to. pushed early last night to get some ci signals.